packages/rpc-api/src/getRecentPrioritizationFees.ts

```diff
@@ -16,17 +15,14 @@ type GetRecentPrioritizationFeesApiResponse = readonly RecentPrioritizationFee[]
 
 export type GetRecentPrioritizationFeesApi = {
     /**
-     * Returns the balance of the account of provided Pubkey
+     * Returns a list of the smallest prioritization fees paid in recent blocks.
```

Comment:
It's nice to have dug and discovered that this is the minimum fee paid by any transaction